app/javascript/dashboard/routes/dashboard/settings/inbox/components/CampaignTable.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <section class="campaigns-table-wrap">
+    <ve-table
```

Comment:
Can we use a normal html table here as we are not using any `data-table` specific features here?